agent/bzero/registration/registration.go

```diff
@@ -21,8 +20,9 @@ const (
 	BZeroConfigStorage    = "BZeroConfig"
 	BZeroRegErrorExitCode = 234
 
-	registrationEndpoint = "api/v2/targets/ssm/register"
-	prodServiceUrl       = "https://cloud.bastionzero.com/" // default
+	registrationEndpoint         = "targets/ssm/register"
```

Comment:
Does this mean that the agent hits either v1 or v2? I think the logic we implemented for simplified registration (aka for the -activationToken or -registrationKey flags) requires the v2 version

Reply:
So this call is proxied through the connection orchestrator now (where the aws region is added). There we are just using the v2 endpoint. 

https://github.com/bastionzero/webshell-backend/blob/45a2b3f3ef643fea841a4264074ffd92f6bc95c2/ConnectionOrchestrator/Controllers/SsmTargetController/SsmTargetController.cs#L42-L54

https://github.com/bastionzero/webshell-backend/blob/45a2b3f3ef643fea841a4264074ffd92f6bc95c2/ConnectionOrchestrator/Bastion/BastionSsmTargetHttpClientService/BastionSsmTargetHttpClientService.cs#L20-L23